pkg/services/provisioning/dashboards/config_reader.go

```diff
@@ -40,7 +40,11 @@ func (cr *configReader) parseConfigs(file os.FileInfo) ([]*DashboardsAsConfig, e
 		}
 
 		if v1 != nil {
-			return v1.mapToDashboardAsConfig(), nil
+			dashboardsAsConfig, err := v1.mapToDashboardAsConfig()
```

Comment:
I think it's better to just return the results directly, since there's no benefit from the extra code:
```suggestion
			return v1.mapToDashboardAsConfig()
```